implementations/rust/ockam/ockam_command/src/status/static/after_long_help.txt

```diff
@@ -0,0 +1,4 @@
+```sh
+# Display information about all of the system's Spaces, Identities, and Nodes.
```

Comment:
I'd simply say `Display information about your Ockam instance and your Orchestrator resources`.

We plan to add more information to the output of this command, so it should be as general as possible, instead of naming the exact objects that it will show.